SMLHelper/Initializer.cs

```diff
@@ -46,28 +41,7 @@ public static void PostPatch()
                 Logger.Error($"Caught exception while trying to initialize SMLHelper{Environment.NewLine}{e}");
             }
         }
-#endif
```

Comment:
In simplifying this class, the log line is now displaying "for Subnautica" on both builds.

Update as needed so the logs print which game this was built for.

Reply:
https://github.com/SubnauticaModding/SMLHelper/pull/126/commits/85f08a8887e1d04052fc264d7889bc02cdcaff42